packages/core/src/types.ts

```diff
@@ -138,6 +138,16 @@ export interface TypeInfo {
   searchParamsDetails?: Record<string, SearchParameterDetails>;
 }
 
+/**
+ * The list of all known search parameter definition bundles
+ * filenames from `@medplum/definitions`.
+ */
+export const SEARCH_PARAMETER_BUNDLE_FILES = [
```

Comment:
Would this make more sense in `@medplum/definitions` - `packages/definitions/src/index.ts`